runtime/include/tt/runtime/runtime.h

```diff
@@ -63,19 +74,34 @@ Device openDevice(DeviceIds const &deviceIds, size_t numHWCQs = 1);
 
 void closeDevice(Device device);
 
-Event submit(Device device, Binary executable, std::uint32_t programIndex,
-             std::vector<Tensor> const &inputs,
-             std::vector<Tensor> const &outputs);
-
 void wait(Event event);
```

Comment:
Since the new submit API returns tensors maybe we should have convenience APIs:
```
void wait(Tensor tensor);
void wait(std::vector<Tensor> const& tensors);
```